src/neural/encoder.cc

```diff
@@ -56,74 +56,74 @@ int ChooseTransform(const ChessBoard& board) {
     return 0;
   }
   auto our_king = (board.kings() & board.ours()).as_int();
-  int transform = 0;
+  int transform = NoTransform;
   if ((our_king & 0x0F0F0F0F0F0F0F0FULL) != 0) {
-    transform |= 1;
+    transform |= Flip;
     our_king = ReverseBitsInBytes(our_king);
   }
   // If there are any pawns only horizontal flip is valid.
   if (board.pawns().as_int() != 0) {
     return transform;
   }
   if ((our_king & 0xFFFFFFFF00000000ULL) != 0) {
-    transform |= 2;
+    transform |= Mirror;
     our_king = ReverseBytesInBytes(our_king);
   }
   // Our king is now always in bottom right quadrant.
   // Transpose for king in top right triangle, or if on diagonal whichever has
   // the smaller integer value for each test scenario.
   if ((our_king & 0xE0C08000ULL) != 0) {
-    transform |= 4;
+    transform |= Transpose;
   } else if ((our_king & 0x10204080ULL) != 0) {
     auto outcome =
         CompareTransposing((board.ours() | board.theirs()).as_int(), transform);
     if (outcome == -1) {
```

Comment:
A reminder that style guide allows writing simple `if `statements which fit one line into one line:
```cpp
if (outcome == -1) return transform;
```

If would save quite a lot of lines, but it's up to you.

Reply:
yeah, I'll make the change.